roles/nginx/tasks/nginx_apt.yml

```diff
@@ -11,7 +9,15 @@
 - name: Add nginx repos
   become: yes
   apt_repository:
-    repo: "deb http://nginx.org/packages/ubuntu/ trusty nginx"
+    repo: "deb http://nginx.org/packages/ubuntu/ {{ ansible_distribution_release|lower }} nginx"
+    state: present
+    update_cache: yes
+  tags: nginx
+
+- name: Add nginx deb-src
```

Comment:
Entire block `deb-src` could be removed for `nginx`. We don't need package sources for sure.
Probably something we forgot to clean before.